invenio_communities/assets/semantic-ui/js/invenio_communities/new.js

```diff
@@ -45,7 +45,7 @@ class CommunityCreateForm extends Component {
               withCredentials: true,
             });
             setSubmitting(false);
-            window.location.href = `/communities/${response.data.id}`;
+            window.location.href = `/communities/${response.data.id}/settings`;
```

Comment:
Hm, should we generate `settings_html` links? (to be extracted into a separate issue of course :) )